server/app/assets/stylesheets/northstar/_uswds-theme-custom-styles.scss

```diff
@@ -89,6 +89,15 @@ See the radio-option below as an example.
   @include u-height(1);
 }
 
+.cf-footer-primary-link {
+  @include u-text('base-darkest');
+  @include u-font-size('body', 'sm');
+  font-style: normal;
```

Comment:
**question:** Do we still need these properties with the u-font-size inclusion?

Reply:
Realized I could use usa-footer__primary-link class